### ISSUE TITLE
Is the project still active?

### ISSUE THREAD

**Issue by liorze** [kubernetes-incubator/kube-mesos-framework#62]
Looks like there haven't been any progress in the past 2-3 months. Is this project still active? Is there anything that we (the devs that have interest in this project) can help with?

@k82cn

**Comment by k82cn** [comment]
@liorze , yes, this's still active; for now, I'm working on the refactor to use k8s's hooks/apis, e.g. scheduler's http extentions, client-go and so on.

If any proposal, please feel free to open an issue :).

**Comment by liorze** [comment]
@k82cn How can I help in this work? Do you track the tasks here or somewhere else?

**Comment by k82cn** [comment]
@liorze , I created some issues for 0.8; if interesting, please @ me in the issue.

The working branch is https://github.com/kubernetes-incubator/kube-mesos-framework/tree/r0.8_working , the TODO list is here: https://github.com/kubernetes-incubator/kube-mesos-framework/projects/1
